nidirect_common/nidirect_common.module

```diff
@@ -266,6 +267,29 @@ function nidirect_common_inline_entity_form_entity_form_alter(&$entity_form, &$f
  * Implements hook_form_BASE_FORM_ID_alter().
  */
 function nidirect_common_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+
+  // Update description for Hide Content flag.
+  if (NestedArray::keyExists($form, ['flag', 'hide_content'])) {
+    $form['flag']['hide_content']['#description'] = t('
+      Check this box to prevent links to this page appearing in navigation menus and landing pages.
+    ');
+  }
+
+  // Hide Featured Content flag.
+  if (NestedArray::keyExists($form, ['flag', 'featured_content'])) {
+    $form['flag']['hide_content']['#access'] = FALSE;
+  }
+
+  // Hide Promotion Options.
+  $form['options']['#access'] = FALSE;
```

Comment:
Can we hide these via the form display settings?

Reply:
Can do but it was 1 line of code Vs updating 17+ Entity Form display configurations.